tools/deploy-transfer-ownership-calibnet.sh

```diff
@@ -0,0 +1,154 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+#####################################
+# Environment variables & defaults  #
+#####################################
+
+: "${FIL_CALIBNET_RPC_URL:?FIL_CALIBNET_RPC_URL not set. Please export it and rerun.}"
+: "${FIL_CALIBNET_PRIVATE_KEY:?FIL_CALIBNET_PRIVATE_KEY not set. Please export it and rerun.}"
```

Comment:
Purely out of curiosity how do you get your private keys?  I ended up needing to write my own script to extract from eth keystore since that design works hard to avoid extraction.